build.gradle.kts

```diff
@@ -101,11 +112,13 @@ subprojects {
       }
     }
 
-    signing {
-      val signingKey: String? by project
-      val signingPassword: String? by project
-      useInMemoryPgpKeys(signingKey, signingPassword)
-      sign(publishing.publications[publicationName])
+    if (!project.hasProperty("skipSigning") || project.property("skipSigning") != "true") {
```

Comment:
adding this allows you to build artifacts locally even without a signing key and password like this

`./gradlew publishToMavenLocal -PskipSigning=true -Pversion={your-local-version-name}`